package.json

```diff
@@ -66,6 +66,8 @@
     "classnames": "^2.2.6",
     "debounce": "^1.2.0",
     "extend": "^3.0.2",
-    "react-stickynode": "^2.1.0"
+    "lodash.debounce": "^4.0.8",
```

Comment:
Looks like we're using two different `debounce`s. Maybe we should pick one or the other?